cmd/validate/image.go

```diff
@@ -277,8 +277,9 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 					// Skip on err to not panic. Error is return on routine completion.
 					if err == nil {
 						res.component.Violations = out.Violations()
+						showSuccesses, _ := cmd.Flags().GetBool("show-successes")
 						res.component.Warnings = out.Warnings()
-						res.component.Successes = out.Successes()
+						res.component.Successes = out.Successes(showSuccesses)
```

Comment:
I think this is better suited for the reporting bit, if we want to keep it here why not do something like:
```suggestion
						if showSuccesses {
							res.component.Successes = out.Successes()
						}
```

Reply:
ack. Updated.